test/integration/expression-tests/collator/accent-equals-de/test.json

```diff
@@ -37,7 +37,17 @@
     "outputs": [true, false],
     "serialized": [
       "case",
-      false,
+      [
+        "==",
+        [
+          "resolved-locale",
+          [
+            "collator",
+            {"caseSensitive": true, "diacriticSensitive": false, "locale": "de"}
```

Comment:
Not related to this change, but these should be spelled `case-sensitive` and `diacritic-sensitive`.